tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -610,8 +666,8 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
     input.map(s => s.split(pattern, limit))
   }
 
-  private def gpuSplit(pattern: String, input: Seq[String], limit: Int): Seq[Array[String]] = {
-    val isRegex = RegexParser.isRegExpString(pattern)
+  private def gpuSplit(pattern: String, input: Seq[String],
+                       limit: Int, isRegex: Boolean): Seq[Array[String]] = {
```

Comment:
nit: indent is off here 

```suggestion
      limit: Int, isRegex: Boolean): Seq[Array[String]] = {
```

Reply:
Fixed by putting parameters on each line per databricks scala style guide.